manifests/b/bitmediae-SolutionsGmbH/ITSClient/3.0.0.0/bitmediae-SolutionsGmbH.ITSClient.locale.de-DE.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: bitmediae-SolutionsGmbH.ITSClient
+PackageVersion: 3.0.0.0
+PackageLocale: de-DE
+Publisher: bit media e-Solutions GmbH
+PackageName: ITS Client
+License: MIT-Lizenz
+ShortDescription: ITS Client

```

```suggestion
ShortDescription: Den ITSR3 Client benötigen alle SchülerInnen, die an einer e-Prüfung teilnehmen möchten.
```
I got it from `file:///C:/Users/WinGet/Downloads/Quickguide_ITSR3_Installation_2019-09-05.pdf`